modules/apps/expando/expando-test/src/testIntegration/java/com/liferay/expando/service/test/ExpandoValueLocalServiceTest.java

```diff
@@ -169,6 +170,32 @@ public void testAddNondefaultLocalizedStringValue() throws Exception {
 		}
 	}
 
+	@Test
+	public void testAddSiteDefaultLocalizedStringValue() throws Exception {
+		Locale originalLocale = LocaleUtil.getSiteDefault();
+
+		LocaleThreadLocal.setSiteDefaultLocale(LocaleUtil.FRANCE);
+
+		ExpandoColumn column = ExpandoTestUtil.addColumn(
+			_expandoTable, "Test Column",
+			ExpandoColumnConstants.STRING_LOCALIZED);
+
+		ExpandoValue value = ExpandoTestUtil.addValue(
+			_expandoTable, column,
+			HashMapBuilder.put(
+				_enLocale, "one"
+			).put(
+				_frLocale, "un"
+			).build());
+
+		value = ExpandoValueLocalServiceUtil.getExpandoValue(
```

Comment:
@crodriguezy Please avoid direct usage of XLocalServiceUtil in integration tests, we can `@Inject` `ExpandoValueLocalService` directly. Thanks!